integration/experiment/energy_efficiency/gpu_activity.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env python3
+#
+#  Copyright (c) 2015 - 2022, Intel Corporation
+#  SPDX-License-Identifier: BSD-3-Clause
+#
+
+'''
+Helper functions for running gpu activity agent experiments.
+'''
+
+import argparse
+import os
+
+import geopmpy.agent
+
+from experiment import launch_util
+from experiment import common_args
+from experiment import machine
+
+def setup_run_args(parser):
+    common_args.setup_run_args(parser)
+
+def report_signals():
+    return []
+
+def trace_signals():
+    return []
+
+def launch_configs(output_dir, app_conf):
+    mach = machine.init_output_dir(output_dir)
+    sys_min = mach.frequency_min()
+    sys_max = mach.frequency_max()
+    sys_sticker = mach.frequency_sticker()
+    config_list = [
+                     {"GPU_FREQ_MAX": float("NAN"),
```

Comment:
Add command line argument to provide GPU_FREQ_MAX and GPU_FREQ_EFFICIENT.   Generate this list from phi 0-1 in 0.1 increments instead of hard-coding